Userland/Libraries/LibWeb/HTML/DragEvent.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2024, Maciej <sppmacd@pm.me>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibWeb/Bindings/DragEventPrototype.h>
+#include <LibWeb/Bindings/Intrinsics.h>
+#include <LibWeb/HTML/DragEvent.h>
+
+namespace Web::HTML {
```

Comment:
Other event classes like this are in the `Web::UIEvents` namespace, does this belong there too?

Reply:
It's defined in the HTML spec: https://html.spec.whatwg.org/multipage/dnd.html#the-dragevent-interface